### PR TITLE
Resolves #235 : Add jsfw documentation

### LINKED ISSUE #235: [ Doc ] missing jsfw in readme

### PR DESCRIPTION
**Pull request by yannis-mlgrn**
Fixes #235 
## Features : 
- 📝 Add jsfw documentation

### CODE REVIEW

File: doc/jsfw/documentation.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+## How install it : 
+
+> clone github repository :
+```bash
+git clone https://github.com/viandoxdev/jsfw.git
+```
+> Go to jsfw directory :
+```bash
+cd jsfw
```
Comment (viandoxdev) [inline doc/jsfw/documentation.md:11]: add 
```bash
git checkout e371ef3ec4053d83abad683289f91a447d9e92ed
```
after cd

**Review by viandoxdev — CHANGES_REQUESTED** [review]

File: doc/jsfw/documentation.md (hunk 1 above)
Comment (yannis-mlgrn) [reply doc/jsfw/documentation.md:11]: fixed here [eb91979](https://github.com/mdl29/donkeycarLPH/pull/237/commits/eb919791a6cdd13f907466166147df454249af17)

**Review by viandoxdev — APPROVED** [review]

**Comment by yannis-mlgrn** [comment]
LGTM !

**Comment by viandoxdev** [comment]
Closed in favor of #242

**Pull request closed without merging**